java/ql/lib/semmle/code/java/security/HttpsUrls.qll

```diff
@@ -30,7 +30,7 @@ class HttpStringLiteral extends StringLiteral {
 abstract class UrlOpenSink extends DataFlow::Node { }
 
 private class DefaultUrlOpenSink extends UrlOpenSink {
-  DefaultUrlOpenSink() { sinkNode(this, "open-url") }
+  DefaultUrlOpenSink() { sinkNode(this, "request-forgery") }
```

Comment:
Should we add a small comment here making clear that `request-forgery` sinks simply make a request to a URL?

Reply:
Added in https://github.com/github/codeql/pull/12916/commits/c50f5885089c2c96be8d6afe9ca9b112f39313bb.
I mimicked the wording in the [documentation](https://github.com/github/codeql/pull/12916/files#diff-58d3e7a419677a7c0cd0e1bfa75f0e45f69269f4a6c4f926c974d0c2234bf599).